skills/meta/writing-skills/SKILL.md

```
@@ -80,7 +85,20 @@ ${SUPERPOWERS_SKILLS_ROOT}
     supporting-file.*     # Only if needed
 ```
 
-**Flat namespace** - all skills in one searchable location
+**Project skills are in your project's `.superpowers/skills/` directory:**
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)